src/Infrastructure/BotSharp.Abstraction/VectorStorage/Models/VectorPayloadValue.cs

```diff
@@ -0,0 +1,58 @@
+using BotSharp.Abstraction.VectorStorage.Enums;
```

Comment:
We can utilize [implict operator](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/operators/user-defined-conversion-operators) to simplier the syntax.
Define the implict opertor:
```csharp
    public static implicit operator byte(Digit d) => d.digit;
    public static explicit operator VectorPayloadValue(byte b) => new VectorPayloadValue(b);
```

Can be used like:
```csharp
        var d = new VectorPayloadValue(7);
        byte number = d;
```